blueocean-config/src/main/java/io/jenkins/blueocean/config/BlueOceanConfigFactory.java

```diff
@@ -0,0 +1,62 @@
+package io.jenkins.blueocean.config;
+
+import javax.annotation.Nonnull;
+
+import hudson.ExtensionPoint;
+import jenkins.model.Jenkins;
+
+/**
+ * Blue Ocean configuration factory which allows different extensions to override the default behavior.
+ * 
+ */
+public abstract class BlueOceanConfigFactory implements ExtensionPoint {
+    public static final String FEATURE_PROPERTY_PREFIX = "blueocean.features.";
```

Comment:
This is a base constant for all the configuration. My intent was to make sure all the configuration starts with the same prefix. Anyway, I can move it.